guide/book.toml

```diff
@@ -5,3 +5,7 @@ author = "PyO3 Project and Contributors"
 
 [preprocessor.pyo3_version]
 command = "python3 guide/pyo3_version.py"
+
+[output.html]
+git-repository-url = "https://github.com/PyO3/pyo3/tree/main/guide"
```

Comment:
Is it customary to link into the tree here instead of just linking the PyO3 repository?

Reply:
mdbook also link into the tree: https://github.com/rust-lang/mdBook/blob/c8db0c8ec65e9ff8e82c580806a7c5dc03774352/guide/book.toml#L13